mdgo/conductivity.py

```diff
@@ -50,19 +56,20 @@ def msd_fft(r):
 
 
 def calc_cond(u, anions, cations, run_start, cation_charge=1, anion_charge=-1):
-    """Calculates the conductivity "mean square displacement" given
-    an MDAnalysis universe (u) and a selection of atoms or molecules (sel)
-
-    Args:
-    u: MDAnalysis Universe
-    sel: Selection of atoms or molecules. Should be
-    a MDAnalysis AtomGroup.
-
-    Returns an array of MSD values for each time in the trajectory.
+    """Calculates the conductivity "mean square displacement" over time
 
     Note:
-        Coordinates must be unwrapped (in dcd file when creating MDAnalysis
-        Universe)
+       Coordinates must be unwrapped (in dcd file when creating MDAnalysis Universe)
+
+    Args:
+        u: MDAnalysis universe
+        anions: MDAnalysis AtomGroup containing all anions (assumes anions are single atoms)
+        cations: MDAnalysis AtomGroup containing all cations (assumes cations are single atoms)
```

Comment:
How much extra work would be required to make this code general enough to consider multi-atom cations and anions?

Reply:
I just made this change! It seems like many of the functionalities in other parts of the code may also assume that ions are single atoms, so we should modify those places as well, and perhaps make the documentation clearer as to how the atom groups should be specified.

Reply:
Thanks @kdfong! And I agree with you that we should try to make other parts of the code more flexible in this regard, too.